utils/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Use an official Python runtime as a parent image
+FROM ubuntu:latest
+
+ENV TZ=Asia/Singapore
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the requirements.txt into the container at /app
+#COPY [ "gtfTobed.py", "/app/" ]
```

Comment:
Will this not be executed? Sorry, I'm not proficient with Dockerfiles, but this seems somehow confusing. If this is commented out, where does the gtfTobed.py come from? :(

Reply:
The `gtfTobed.py` will come from the line `COPY . /app`